app/experimenter/experiments/management/commands/generate_dashboards.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.conf import settings
+
+from redash_client.client import RedashClient
+from redash_client.dashboards.StatisticalDashboard import (
+    StatisticalDashboard)
+
+from django.core.management.base import BaseCommand
+from experimenter.experiments.models import Experiment
+
+
+class Command(BaseCommand):
+    POPULATION_TEMPLATE = 'UT Experiment Template: Population Size'
+    EVENTS_PER_HOUR_TEMPLATE = 'TTests Template Per Hour UT Five:'
+    UT_HOURLY_TTABLE = "Statistical Analysis (Per Active Hour) - UT"
+    help = 'Generates Redash dashboards'
+
+    def generate_dashboards(self):
+        redash_client = RedashClient(settings.REDASH_API_KEY)
+
+        relevant_experiments = Experiment.objects.filter(
+            status__in=(
+                Experiment.STATUS_LAUNCHED,
+                Experiment.STATUS_COMPLETE))
+        for exp in relevant_experiments:
+            try:
+                dash = StatisticalDashboard(
+                  redash_client,
+                  settings.AWS_ACCESS_KEY,
+                  settings.AWS_SECRET_KEY,
+                  settings.S3_BUCKET_ID_STATS,
+                  exp.project.name,
+                  exp.name,
+                  exp.slug,
+                  start_date=exp.start_date.strftime("%Y-%m-%d")
```

Comment:
Should we also pass end_date for experiments that are over?

Reply:
Yes, thank you. Separately, I think we need some logic for only updating dashboards for X amount of time after the end of an experiment. Otherwise we end up with experiments that have ended a month ago and we are still attempting to update the dashboards. I'll file this as a separate issue though.